app/cmd/controller.go

```diff
@@ -96,9 +96,10 @@ func startController(c *cli.Context) error {
 	if c.NArg() == 0 {
 		return errors.New("volume name is required")
 	}
-	name := c.Args()[0]
+	volumeName := c.Args()[0]
+	// The global "--volume-name" flag is ignored here. It is redundant with the above required positional argument.
```

Comment:
I am thinking if we can use the global "--volume-name" flag here as well

Reply:
I think it is a bit awkward either way, but prefer to leave it as is because:

1. `arg[0]` is a required parameter, while generally `--volume-name` is not.
1. `arg[0]` has been a part of the CLI API from (at least close to) the beginning and the information it contains is still relevant/necessary.

WDYT?